### PR TITLE
first dungeon probably finished

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

2 more items establishing the raw basis of lore while also explaining literally nothing

## Why It's Good For The Game

if people find this i will be actually shocked tbh

### CODE REVIEW

**Comment by EaglePhntm** [comment]
mm dungeon

File: modular_hearthstone/code/game/objects/structures/fluff.dm [hunk 1]
```diff
@@ -39,3 +39,32 @@
...
+	icon = 'modular_hearthstone/icons/roguetown/misc/64x64.dmi'
+	icon_state = "answer"
+	layer = 3
+
+/obj/structure/fluff/statue/answer/examine(mob/user)
+	. = ..()
+	if(!user.is_literate())
+		. += "My head hurts."
+	else
+		. += "A plucked gaze at a furthest distance; the eye of God-Things. Psydon died in this cradle, torn apart by hubris and split into his children."
```
Comment (EaglePhntm) [inline modular_hearthstone/code/game/objects/structures/fluff.dm:70]: everyone is literate after a pr now, nobody will see this.

**Comment by bearrrrrrrr** [comment]
> But in seriousness, what is your actual angle here. I'm getting a slight copied media feeling

copied from?

File: modular_hearthstone/code/game/objects/structures/fluff.dm (hunk 1 above)
Comment (bearrrrrrrr) [reply modular_hearthstone/code/game/objects/structures/fluff.dm:71]: this is a good point. i'll implement an advanced literacy proc in the next PR, i want to try to keep my pushes (somewhat) atomized

**Comment by bearrrrrrrr** [comment]
> At some point. All devil-shit media blends together. Honestly, I thought you were copying, this, https://arcanum.fandom.com/wiki/Vendigroth

oooh this looks neat frankly but no, i've never seen that in my life. definitely gonna waste some time looking at it though lmao ty

as for the direction/slotting stuff in: i literally am wedded to absolutely nothing besides the general overview of 'things were good, Breach:tm: happened, things are bad now', with no further holdups. there's definitely a greater convo to be had here regarding lore and stuff so feel free to hmu on the discord or wherever if you have any ideas :3

**Comment by bearrrrrrrr** [comment]
> [loretext]

👁️ 

definitely including that in the next pr as well

**Pull request merged**